database/buildSchema/37_review_list.sql

```diff
@@ -0,0 +1,37 @@
+-- Aggregated VIEW method of all data review data, use in application_list for application list page
+CREATE FUNCTION review_list (reviewerid int)
+    RETURNS TABLE (
+        application_id int,
+        review_available_for_self_assignment_count bigint,
+        review_assigned_count bigint,
+        review_assigned_not_started_count bigint,
+        review_draft_count bigint,
+        review_submitted_count bigint,
+        review_change_request_count bigint,
+        review_pending_count bigint
+    )
+    AS $$
+    SELECT
+        review_assignment.application_id AS application_id,
+        COUNT(*) FILTER (WHERE review_assignment.status = 'Available for self-assignment') AS review_available_for_self_assignment_count,
+        COUNT(*) FILTER (WHERE review_assignment.status = 'Assigned') AS review_assigned_count,
```

Comment:
Maybe better `review_assigned_started_count` instead? If we need the total we can sum up both

Reply:
would that be review_draft_count ?

That would make the following actions:

Start -> review_assigned_not_started_count
Re-Review-> review_pending_count
Update -> review_change_request_count
Continue -> review_draft_count
Self-Assign -> review_available_for_self_assignment_count

review_assigned_count -> not really sure what to indicate as action, but the idea is it can be filtered by it

Re-Review and Update would change to 'Continue' in application list (but would be correctly represented in review_home

Reply:
Got it. Thanks for the explanation : ) I think this would be very useful to add to our documentation about showing actions (if we already have one in progress otherwise to be done as part of separated issue) #474